app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -73,6 +75,17 @@ public function form(Form $form): Form
     {
         return $form
             ->schema([
+                Section::make('Application Name')
+                    ->aside()
+                    ->schema([
+                        TextInput::make('application_name')
+                            ->label(false)
+                            ->maxLength('255')
+                            ->model(
+                                ThemeSettings::getSettingsPropertyModel('theme.application_name'),
```

Comment:
I don't think the application name needs a model as its just a settings property & not media